cli/pkg/commands/images/describe.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2024. Open Data Hub Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	"fmt"
+
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = common.NewCmd(
+	fmt.Sprintf(
+		"describe -%s model-id -%s version [-%s model-registry-url] [-%s params-file]",
+		flags.FlagModelID.Shorthand(),
+		flags.FlagVersionName.Shorthand(),
+		flags.FlagModelRegistryURL.Shorthand(),
+		flags.FlagParams.Shorthand(),
+	),
+	"Show details of an model image along with its params",
+	`Show details of an model image along with its params
+	This command allows you to check details about the model image which can be accessed by passing 
+	model image id and model version name as arguments to the command
```

Comment:
This one appears to have different formatting than what @devguyio has done for the others. E.g. https://github.com/opendatahub-io/ai-edge/blob/main/cli/pkg/commands/images/images.go#L213-L215

Reply:
changed this too